test/enhance/test_adjust.py

```diff
@@ -599,3 +599,150 @@ def test_gradcheck(self, device):
         img = utils.tensor_to_gradcheck_var(img)  # to var
         assert gradcheck(kornia.adjust_brightness, (img, 2.),
                          raise_exception=True)
+
+
+class TestEqualize:
+    def test_shape_equalize(self, device):
+        bs, channels, height, width = 1, 3, 4, 5
+
+        inputs = torch.ones(channels, height, width)
```

Comment:
pytorch recommendation to initialize tensors:
`inputs = torch.ones(channels, height, width, device=device, dtype=dtype)`